kubectl-cloudflow/config/config.go

```diff
@@ -303,10 +307,34 @@ func validateConfig(config *Config, applicationSpec cfapp.CloudflowApplicationSp
 	return nil
 }
 
+func validateLabels(podConfig *configuration.Config, podName string) error {
```

Comment:
See https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set there are some more rules we could validate I think.

Reply:
Done